Mollier/SAM.Core.Mollier.UI/Classes/DisplayAnalyticalObject.cs

```diff
@@ -0,0 +1,236 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace SAM.Core.Mollier.UI
+{
+    public class DisplayUIMollierObject
+    {
+        public IUIMollierObject UIMollierObject;
+        public int Index;
+        private double airFlow;
+        private string airFlowUnit;
+
+        public DisplayUIMollierObject(UIMollierPoint uIMollierPoint)
+        {
+            UIMollierObject = uIMollierPoint;
+        }
+
+        public DisplayUIMollierObject(UIMollierProcess uIMollierProcess, int index, double airFlow, string airFlowUnit)
```

Comment:
Do not use string for airFlowUnit . Use SAM.Units.UnitsType instead:
SAM.Units.Query.Abbreviation(unitType) will return m3/s
Add missing UnitTypes if necessary